.github/workflows/export_strings.yml

```diff
@@ -1,5 +1,12 @@
 name: Import strings
 on:
+  schedule:
+    - cron: '0 16 * * 3' # At 9:00 PDT on Wednesday
+    inputs:
```

Comment:
I completely missed this (schedule doesn't support inputs). Fixed directly on main.

Reply:
Oh, that seems like a strange design constraint, but good to know. I see that the script is being run with `type ""` argument and executing successfully, so guessing if an empty value is provided the script resolves to standard?

Reply:
Well, you can't pass a value to the cronjob at run-time, so it makes sense. What was very confusing is that the error pointed at a wrong format on line 3 (cron).

And yes, the script [has a default value of standard](https://github.com/mozilla-l10n/focusios-l10n/blob/main/.github/scripts/update_other_locales.py#L57).

Reply:
Ah, thanks for this. Helps refine my mental model of github actions.